### PR TITLE
database based on a diagram

### PR DESCRIPTION
**Pull request by Caren-Koroeny**
-  <h1> Hello Reviewers </h1>
In this branch, @hakichris and @Caren-Koroeny created tables based on database diagram. 
- [x] correct Gitflow
- [x]  Tables created

### CODE REVIEW

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @Caren-Koroeny @hakichris , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 


## Highlights
- [ ] The table has been created successfully  with all the connexion set :+1: 
- [ ] Descriptive PR `title` and `summary` :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
soclose

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+); 
+
+CREATE TABLE medicalhistorytreatment (
+medical_history_id int NOT NULL,
+treatment_id int NOT NULL, 
+CONSTRAINT fk_medicalhostories FOREIGN KEY (medical_history_id) 
+	REFERENCES medical_histories(id), 
+CONSTRAINT fk_treatments FOREIGN KEY (treatment_id) 
+	REFERENCES treatments(id)
+);
```
[the remark this answers is not in the document]
Comment (hakichris) [reply schema_based_on_diagram.sql:29]: Thank you @christianbiring1 , we have rectified it.

File: schema_based_on_diagram.sql [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+unit_price decimal, 
+quantity int, 
+total_price decimal,
+invoice_id int, 
+treatment_id int,
+CONSTRAINT fk_invoice_treatment FOREIGN KEY (treatment_id)
+	REFERENCES treatments (id),
+CONSTRAINT fk_invoiceItems_invoices FOREIGN KEY (invoice_id)
+	REFERENCES invoices (id)
+);
```
[the remark this answers is not in the document]
Comment (hakichris) [reply schema_based_on_diagram.sql:52]: Thank you @christianbiring1 , we have rectified it.

**Review by OyePriscilla — APPROVED** [review]
Hi @Caren-Koroeny and @hakichris 👏 👏, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Create `sql` file to implement database from the given diagram ✔️ 
Implement `Join tables` from many-to-many relationships ✔️ 
Add the FK indexes ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**